contracts/BaseStrategy.sol

```diff
@@ -182,6 +182,8 @@ abstract contract BaseStrategy {
 
     event UpdatedDebtThreshold(uint256 debtThreshold);
 
+    event EnabledEmergencyExit();
```

Comment:
Can we do `EmergencyExitEnabled()` instead?

Reply:
fixed in latest